utils/_context/_scenarios/__init__.py

```diff
@@ -76,6 +76,7 @@ class _Scenarios:
             "DD_TRACE_COMPUTE_STATS": "true",
             "DD_TRACE_FEATURES": "discovery",
             "DD_TRACE_TRACER_METRICS_ENABLED": "true",  # java
+            "DD_TRACE_DEBUG": "true",
```

Comment:
You may find some info here : https://github.com/DataDog/system-tests/blob/main/docs/execute/dd-trace-debug.md